chapters/annotations.tex

```diff
@@ -1254,16 +1256,15 @@ \section{Graphical User Interface}\label{annotations-for-the-graphical-user-inte
 
 This section describes the annotations that are used to define properties of the graphical user interface.
 
-\begin{lstlisting}[language=modelica]
-annotation(preferredView = $\mathit{view}$)
+\begin{lstlisting}[language=modelica, mathescape=true]
+String preferredView = $\mathit{view}$;
 \end{lstlisting}
 
 The \fmtannotationindex{preferredView} annotation defines the default view when selecting the class.
 The $\mathit{view}$ is a \lstinline!String! literal where \lstinline!"info"! means class documentation (``information''), \lstinline!"diagram"! means diagram view, \lstinline!"icon"! means icon view, and \lstinline!"text"! means Modelica source code (``text'').
 
-\begin{lstlisting}[language=grammar]
-  documentation-class-annotation:
-     annotation "(" DocumentationClass "=" true ")"
+\begin{lstlisting}
+Boolean DocumentationClass = true;
```

Comment:
Is it allowed to specify ``DocumentationClass = false``? The description assumes that if it was specified it is specified as true. Also giving it a binding equation like that assumes that there is a default value, so it would be sufficient to specify: ``annotation(DocumentationClass)``. Is that indeed allowed?

Reply:
I believe that a good solution would be to remove that binding and change the text for DocumentationClass to start with: "If true..." and possibly add "if false it has no impact"

Reply:
Have now added that.